docs/entities-intro.rst

```diff
@@ -42,45 +42,41 @@ Entities can also be helpful for sharing data that rarely changes between multip
+A "case" is a term used in many domains to mean an instance of something. A case is opened when a specific condition is detected, may go through multiple steps across a broad range of caseworkers while the case condition is still true, and then is closed when the case condition is no longer true. For example:
 
-* Law: legal teams may manage human rights violation cases or property damage cases.
+* Medicine: healthcare workers may manage pregnancy cases
 
-Other related terms include "issues" or "problems". An "incident" generally references to a group of cases with the same root cause. For example, an electricity provider may get multiple case reports of households without electricity and declare an outage incident that needs to be managed.
+* Utilities: a electricity provider may get multiple case reports of households without electricity
 
-You can use ODK Entities to support managing cases, issues, problems, incidents, etc. To do so, you will likely have Entity Lists with names like ``malaria_cases``, ``outages``, ``issues``. 
+* Crisis response: a humanitarian team may be involved in child protection cases or asylum cases
 
 .. seealso::
     The :doc:`Community reporting tutorial <tutorial-community-reporting>` implements a simple case management workflow in which anyone can open a case (called "problem" in the tutorial) and specific individuals can resolve them.
 
-While case management is a process that is familiar in many fields, it's not universal. ODK has always been a generic tool and by using more a more general approach, we believe we can better support the needs of users who work in diverse domains.
+You can use Entities to support managing pregnancies, power outages, legal cases, and or other case management tasks. And, you can use Entity Lists with terminology that matches you your specific workflow like ``risky_pregnancies``, ``power_outages``, and ``vulnerable_children``. 
 
-Even in fields where case management is common, there is often a need to support other kinds of workflows within the same tool and it can be awkward to use the word "case" in those contexts, especially when referencing concrete entities such as participants or hospitals. Our goal is to let you define Entity Lists that make sense in your context and to allow you to use and connect them in ways that best support your workflow.
+We use the word "entities" because Entities are much more flexible. They don't impose any requirement that you to open or close a case. Also, even in domains where case management is common, there is often a need to support other kinds of workflows within the same tool and it can be awkward to use the word "case" in those contexts, especially when referencing real things like trees, people, or cities. Our goal is to let you define Entity Lists that make sense in your context and to allow you to use and connect them in ways that best support your workflow.
```

Comment:
I think "much more flexible" isn't quite right. We use the word "Entities" because it's a more generic framing.

"They don't impose any requirement that you to open or close a case." Not sure how strong this is.